test/NCalc.Tests/Lambdas.cs

```diff
@@ -242,5 +242,21 @@ public void ShouldHandleDataConversions(string input, bool expected)
 
             Assert.Equal(expected, sut(context));
         }
+
+        [Theory]
+        [InlineData("Min(3,2)")]
+        [InlineData("Min(3.2,6.3)")]
+        [InlineData("Max(2.69,9.6)")]
+        [InlineData("Max(9,6)")]
+        [InlineData("Pow(5,2)")]
+        [InlineData("Pow(5,0.36)")]
+        public void ShouldHandleNumericBuiltInFunctions(string input)
+        {
+            var expression = new Expression(input);
+            var sut = expression.ToLambda<object>();
+            Assert.NotNull(sut());

```

Comment:
can you assert the correct return value here? e.g. for Min(3,2) it should equal 2